### PR TITLE
Add AnimeGDRClub

### LINKED ISSUE #6088: [Source Request] <Anime GDR Club>
## Source website - agcscanlation.it
- Name: Anime GDR Club
- Link: animegdrclub.forumfree.it - agcscanlation.it
- Language(s): Italian

## Other details
Forum:
Screenshot_3

Reader:
Screenshot_4

### PR DESCRIPTION
**Pull request by ReLoia**
Closes #6088

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            pages.add(Page(i, "", baseUrl + it.attr("src")))
+        }
+
+        return pages
+    }
+
+    override fun imageUrlParse(document: Document) = ""
+    override fun imageRequest(page: Page): Request {
+        val imgHeader = Headers.Builder().apply {
+            add("User-Agent", "Mozilla/5.0 (Linux; U; Android 4.1.1; en-gb; Build/KLP) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Safari/534.30")
```
Comment (arkon) [inline src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:193]: This looks like a really old user agent?

File: src/it/animegdrclub/build.gradle [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
```
Comment (arkon) [inline src/it/animegdrclub/build.gradle:3]: Don't need this if you're not using kotlinx-serialization.

File: src/it/animegdrclub/build.gradle [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+ext {
+    extName = 'Anime GDR Club'
+    pkgNameSuffix = 'it.animegdrclub'
+    extClass = '.AnimeGDRClub'
+    extVersionCode = 3
```
Comment (arkon) [inline src/it/animegdrclub/build.gradle:9]: Why is it starting at 3?

Comment (ReLoia) [reply src/it/animegdrclub/build.gradle]: Sorry, I made this extension using another one as base and i completely forgot about changing the version and imports. 
Resolved in the latest review

File: src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt (hunk 1 above)
Comment (ReLoia) [reply src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:192]: The extension works fine with this user agent. Should I change it? I've seen other extensions using the same user agent or similar so I used that one..

Comment (alessandrojean) [reply src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:192]: Yes, you should always use updated User-Agent strings. Extensions that still have outdated ones are simply not updated by any contributor yet.

Comment (ReLoia) [reply src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:192]: Okay, thanks for the information. One of [those](https://www.whatismybrowser.com/guides/the-latest-user-agent/android) should be fine? Or should I use one of an older android version like this one that I've got from `Network conditions` tab on Chrome DevTools: `Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.115 Mobile Safari/537.36`?

Comment (alessandrojean) [reply src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:192]: You can use one from the website you mentioned, and you can also use an Desktop one if you prefer. Alternatively you can just don't provide any User-Agent and the extension will automatically use the one inside the app (that is a Desktop browser).

Comment (ReLoia) [reply src/it/animegdrclub/src/eu/kanade/tachiyomi/extension/it/animegdrclub/AnimeGDRClub.kt:192]: Okay, thank you.
Resolved in the latest review

**Pull request merged**